pyproject.toml

```
@@ -122,8 +122,8 @@ dev = [
     "ruff==0.3.5",
 ]
 upgrade-extension = [
-    "pyyaml-include<2.0",
-    "copier>=8,<10",
+    "pyyaml-include<3.0",
+    "copier>=9,<10",
     "jinja2-time<0.3",
     "pydantic<2.0",
     "tomli-w<2.0"
```

Should removing those two be enough or just changing the pydantic constraint? There are no failure on main due to `pydantic<3.0`.

```suggestion
```

---

I think it is safer to keep a pin for dependencies which do know that may break things when they release a new version; let's just bump pydantic constraint to match with the `main`:

```suggestion
    "pydantic<3.0",
    "tomli-w<2.0"
```